src/main/java/io/github/nubesgen/configuration/NubesgenConfiguration.java

```diff
@@ -170,6 +170,11 @@ public boolean isRuntimeNodejs() {
         return RuntimeType.NODEJS.equals(this.getRuntimeType());
     }
 
+    @JsonIgnore
+    public boolean isRuntimeDefault() {
+        return !isRuntimeSpring();
```

Comment:
This will endup being `return !isRuntimeSpring() && !isRuntimeQuarkus() && !isRuntimeMicronaut();`